### ISSUE TITLE
[Suggestion] API Key as a secret

### ISSUE THREAD

**Issue by tmosleyIII** [NFT-Worlds/WRLD-Payments-API#3]
The API keys can be turned into a secret and then substituted with an additional build step.

```
on:
  push:
    branches:
      - main
    tags:
      - v*
  pull_request:

jobs:
build:
    runs-on: ubuntu-latest
    steps:
    - name: Checkout
      uses: actions/checkout@v2
    - name: Substitution
      uses: microsoft/variable-substitution@v1 
    with:
        files: 'config.yml'
    env:
        polygon_https_rpc: ${{ secrets.POLYGON_HTTPS_RPC }}
        ethereum_https_rpc: ${{ secrets.ETHEREUM_HTTPS_RPC }}
```

**Comment by iamarkdev** [comment]
@Jaxkr tagged you for consideration & your thoughts.

**Comment by Jaxkr** [comment]
Resolved alongside #10
